data/human/deep jobs.txt

```diff
@@ -353,3 +353,616 @@ mission "Record academic conference"
+	cargo "secure package" 1
+	to offer
+		random < 20
+		not "deep security reveal"
+		"reputation: Deep Security" >= -50
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 50000 10000
+		"reputation: Deep Security" += 0.5
+		"secure deep package" ++
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+mission "Secure Transport Job (Secret) [3]"
+	name "Secure delivery to <planet>"
+	job
+	repeat
+	description "Deliver a secure package of supplies to <destination>. Payment is <payment>."
+	cargo "secure package" 1
+	to offer
+		random < 20
+		not "deep security reveal"
+		"reputation: Deep Security" >= -50
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 50000 10000
+		"reputation: Deep Security" += 0.5
+		"secure deep package" ++
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+mission "Secure Transport Job (Secret) [4]"
+	name "Secure delivery to <planet>"
+	job
+	repeat
+	description "Deliver a secure package of supplies to <destination>. Payment is <payment>."
+	cargo "secure package" 1
+	to offer
+		random < 20
+		not "deep security reveal"
+		"reputation: Deep Security" >= -50
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 50000 10000
+		"reputation: Deep Security" += 0.5
+		"secure deep package" ++
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+
+mission "Deep Security Package Reveal"
+	minor
+	invisible
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	to offer
+		"reputation: Deep Security" >= 100
+		"secure deep package" >= 30
+		"mystery cube" >= 5
+		not "deep security reveal"
+		or
+			not "chosen sides"
+			has "main plot completed"
+	on offer
+		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`
+		conversation
+			`As you walk away from a food stand in the spaceport a rather young man in active wear steps away from the wall and smoothly sidles up next to you between one step and the next.`
```

Comment:
What does 'between one step and the next' mean in this context?

Reply:
It means literally what it says.